src/components/FormSteps/AccountInfo.vue

```diff
@@ -472,14 +503,30 @@ export default{
             10000
           );
 
+          e = e.response;
+            if (e.status === 422)
+                this.error = e.data.errors ? e.data.errors : e.data;
+            Flash.setError(
```

Comment:
This should work here

Flash.setError(e.status === 422 ? this.$displayErrorText(e?.data?.errors) : e.message,10000);

Reply:
I think we don't need the error, bvn error is being handled another way, I can just remove what I had before

Reply:
i don't understand that part of the code, use optional chaining instead of ternary